posts/2019/11/Variable-Explorer.md

```diff
@@ -0,0 +1,73 @@
+<!--
+.. title: Variable Explorer improvements in Spyder 4
+.. slug: Variable-Explorer-improvements-in-Spyder 4
+.. date: 2019-11-12 12:00:00 UTC-05:00
+.. author: Daniel Althviz
```

Comment:
@juanis2112 since you're working on this PR, I would think you're at least a co-author together with @dalthviz?

Reply:
Didn't even notice that, heh! But very much this.

Reply:
@rgommers I actually only translated Daniel's work. It's fine by me that he's the author.